README.md

```diff
@@ -31,85 +31,93 @@ The service is subscribed to the following [Keptn CloudEvents](https://github.co
-    --from-literal="DT_TENANT=<DT_TENANT>" 
-    --from-literal="KEPTN_API_URL=<KEPTN_API_URL>" 
-    --from-literal="KEPTN_API_TOKEN=<KEPTN_API_TOKEN>"
-    -oyaml --dry-run=client | kubectl replace -f -
-    ```
+### 2. Create a Secret with Credentials
 
-**Deploy the Service:**
-* The `dynatrace-service` supports to automatically generate tagging rules, problem notifications, management zones, dashboards, and custom metric events in your Dynatrace tenant.
- You can configure whether these entities should be generated within your Dynatrace tenant by the environment variables specified in the provided [values.yml](https://raw.githubusercontent.com/keptn-contrib/dynatrace-service/$VERSION/chart/values.yaml),
- i.e., using the variables `dynatraceService.config.generateTaggingRules` (default `false`), `dynatraceService.config.generateProblemNotifications` (default `false`), `dynatraceService.config.generateManagementZones` (default `false`), `dynatraceService.config.generateDashboards` (default `false`), `dynatraceService.config.generateMetricEvents` (default `false`), and `dynatraceService.config.synchronizeDynatraceServices` (default `true`).
- 
-* The `dynatrace-service` by default validates the SSL certificate of the Dynatrace API.
-If your Dynatrace API only has a self-signed certificate, you can disable the SSL certificate check
-by setting the environment variable `dynatraceService.config.httpSSLVerify` (default `true`) specified in the [values.yml](https://raw.githubusercontent.com/keptn-contrib/dynatrace-service/$VERSION/chart/values.yaml) to `false`.
+Create a secret (named `dynatrace` by default) containing the credentials for the Dynatrace Tenant (`DT_API_TOKEN` and `DT_TENANT`) and optionally for the Keptn API (`KEPTN_API_URL`, `KEPTN_API_TOKEN` and `KEPTN_BRIDGE_URL`).
 
-* The `dynatrace-service` can be configured to use a proxy server via the `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables  as described in [`httpproxy.FromEnvironment()`](https://golang.org/pkg/vendor/golang.org/x/net/http/httpproxy/#FromEnvironment). As the `dynatrace-service` connects to a `distributor`, a `NO_PROXY` entry including `127.0.0.1` should be used to prevent these from being proxied. The `HTTP_PROXY` and `HTTPS_PROXY` environment variables can be configured using the `dynatraceService.config.httpProxy` (default `""`) and `dynatraceService.config.httpsProxy` (default `""`) in [values.yml](https://raw.githubusercontent.com/keptn-contrib/dynatrace-service/$VERSION/chart/values.yaml), `NO_PROXY` is set to `127.0.0.1` by default. For example:
+```console
+kubectl -n keptn create secret generic dynatrace \
+--from-literal="DT_API_TOKEN=$DT_API_TOKEN" \
+--from-literal="DT_TENANT=$DT_TENANT" \
+--from-literal="KEPTN_API_URL=$KEPTN_API_URL" \
+--from-literal="KEPTN_API_TOKEN=$KEPTN_API_TOKEN" \
+--from-literal="KEPTN_BRIDGE_URL=$KEPTN_BRIDGE_URL" \
+-oyaml --dry-run=client | kubectl replace -f -
+```
```

Comment:
this will need to change once  https://github.com/keptn-contrib/dynatrace-service/issues/309 is implemented, right?